src/main.rs

```diff
@@ -33,9 +35,20 @@ Options:
   -n, --bench     Run `cargo bench` when a file is modified
 
 Default options are `build` and `test`
-");
+";
 
-#[derive(Copy)]
+#[derive(RustcDecodable, Debug)]
+struct Args {
+      flag_build: bool,
+      flag_doc: bool,
+      flag_test: bool,
+      flag_bench: bool,
+    //flag_help: _, // These three flags are not used. They are generated by docopt
```

Comment:
Hmm, `flag_help` is definitely not needed. Both -h and --help work without it. It was only needed before because docopt_macros was defining it in the autogenerated Args struct. Since I removed docopt_macros (because it is incompatible with stable) I defined the struct explicitly, so there is no need to include unneeded fields.

As I expected, adding or removing `cmd_cargo` and `cmd_watch` doesn't seem to make any difference. I'm glad you asked me to check though, because `cargo watch` does not work for me either way.  (even with `flag_help`) :/

All I'm getting is

    $ cargo watch
    Invalid arguments.
    
    Usage: cargo-watch [options]
           cargo watch [options]

`strace` shows cargo-watch is getting invoked as

    execve("/home/aij/bin/cargo-watch", ["/home/aij/bin/cargo-watch", "watch", "-h"], [/* 41 vars */]) = 0

If it used to work before, I'm guessing the breakage is due to some change to either `cargo` or `docopt`. I'll have to look into it some more.